package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "nodemon index.js",
     "eslint": "eslint .",
     "eslint:fix": "eslint . --fix",
-    "test": "mocha --watch tests/*.spec.js tests/**/*.spec.js"
+    "test": "mocha --exit tests/*.spec.js tests/**/*.spec.js"
```

Comment:
Ideally, the `--exit` flag should not be used (as recommended by the mocha docs). The tests should clean up themselves. I guess for now let's use it, then tidy up the tests later.